fmriprep/tests/test_config.py

```diff
@@ -0,0 +1,41 @@
+"""Check the configuration module and file."""
+from pathlib import Path
+from pkg_resources import resource_filename as pkgrf
+from toml import loads
+from niworkflows.utils.spaces import format_reference
+
+from .. import config
+
+
+def test_config_spaces():
+    """Check that all necessary spaces are recorded in the config."""
+    filename = Path(pkgrf('fmriprep', 'data/tests/config.toml'))
+    settings = loads(filename.read_text())
+    for sectionname, configs in settings.items():
+        if sectionname != 'environment':
+            section = getattr(config, sectionname)
+            section.load(configs, init=False)
+    config.nipype.init()
+    config.loggers.init()
+    config.init_spaces()
+
+    spaces = config.workflow.spaces
+    assert "MNI152NLin6Asym:res-2" not in [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/poldracklab/fmriprep/pullRequest?prid=5112966)